Purchases/Purchasing/OfferingsFactory.swift

```diff
@@ -35,8 +35,26 @@ class OfferingsFactory {
         return Offerings(offerings: offerings, currentOfferingID: currentOfferingID)
     }
 
+    func createOfferings(withProducts products: [String: ProductDetails],
+                         data: [String: Any]) -> Offerings? {
+        let productIdentifiersAndDetailsAsTuple = products.map { productIdentifier, productDetails in
```

Comment:
isn't this just re-creating the same dictionary?

Reply:
lol what was I thinking